mne/event.py

```diff
@@ -827,3 +828,289 @@ def concatenate_events(events, first_samps, last_samps):
         events_out = np.concatenate((events_out, e2), axis=0)
 
     return events_out
+
+
+class Elekta_event(object):
+    """ Represents a trigger event in Elekta/Neuromag system.  """
+    # original DACQ variable names
+    vars = ['Name', 'Channel', 'NewBits', 'OldBits', 'NewMask',
+            'OldMask', 'Delay', 'Comment']
+
+    def __init__(self, name, channel, newbits, oldbits, newmask,
+                 oldmask, delay, comment):
+        self.name = name  # short name for the event
+        self.channel = channel  # stimulus channel, e.g. STI101
+        self.newbits = int(newbits)  # state after trigger transition
+        self.oldbits = int(oldbits)  # state before trigger transition
+        self.newmask = int(newmask)  # bitmask for post-transition state
+        self.oldmask = int(oldmask)  # bitmask for pre-transition state
+        self.delay = float(delay)  # delay to stimulus (s)
+        self.comment = comment  # verbose comment for the event
+        # non-DACQ vars
+        self.index = None  # index of event in DACQ list
+        self.in_use = False  # whether event is referred to by a category
+
+    def __repr__(self):
+        s = '<Elekta_event | '
+        s += 'index: %d ' % self.index
+        s += 'name: "%s", ' % self.name
+        s += 'comment: "%s", ' % self.comment
+        s += 'pre-state: %d, ' % self.oldbits
+        s += 'pre-mask: %d, ' % self.oldmask
+        s += 'post-state: %d, ' % self.newbits
+        s += 'post-mask: %d, ' % self.newmask
+        s += 'delay: %.3f (s)>' % self.delay
+        return s
+
+
+class Elekta_category(object):
```

Comment:
(In terms of a nice string representation, then this could be a simple wrapper around `dict` where you only add a `__repr__` method -- there are many advantages to using a thin wrapper around a built-in type)